modules/plot/test/spec.coffee

```diff
@@ -1,8 +1,12 @@
+# Used to mimic an event call for a node
+fakeNodeEvent = (node, eventName) ->
+  if node?
+    (e) -> hx.select.getHexagonElementDataObject(node).eventEmitter?.emit((if eventName? and isNaN(eventName) then eventName else 'click'), e)
```

Comment:
also I can envisage this being useful in multiple tests. Might it be worth having a "test helper" module ?